projects/templates/src/lib/components/po-page-blocked-user/po-page-blocked-user-contacts/po-page-blocked-user-contacts.component.ts

```diff
@@ -34,31 +41,19 @@ export class PoPageBlockedUserContactsComponent implements AfterViewInit, OnChan
 
   constructor(private changeDetector: ChangeDetectorRef) {}
 
-  ngAfterViewInit() {
-    this.checkContactItemWidth();
-    this.changeDetector.detectChanges();
-  }
+  private checkContactItemWidth() {
+    this.overflowItem = true;
 
-  ngOnChanges(changes: SimpleChanges) {
-    if (changes.email || changes.phone) {
-      this.checkContactItemWidth();
-    }
-  }
+    if (this.phone && this.email) {
+      this.changeDetector.detectChanges();
```

Comment:
Com o changedetecction normal não é preciso forçar a detecção

Reply:
É necessário o changeDetection aí para manipulação de dom. Sem ele não é possível detectar a largura do elemento para o cálculo da colunagem. Outra solução seria o setTimeout que já estava presente.